lib/uipage/landing.dart

```diff
@@ -511,8 +511,8 @@ class _LandingPageState extends State<LandingPage> with TickerProviderStateMixin
                  Navigator.pushReplacementNamed(context, '/search');
                 },
               ),
-              IconButton(
-                icon: Icon(Icons.help_outline, semanticLabel: 'help',),
+              FlatButton(
```

Comment:
why was this chnaged?

Reply:
Gino said that the icon was not serving the purpose to a normal user and suggested I change it.